portal/server/controllers/dashboard/deals/index.js

```diff
@@ -119,10 +119,11 @@ exports.allDeals = async (req, res) => {
   const { user } = req.session;
   const currentPage = req.params.page;
 
-  let activeSortByOrder = CONSTANTS.SORT_BY.DEFAULT;
+  let activeSortByOrder = req.session.sortBy;
```

Comment:
Need a validation for session to exists, if no session exists then need `SORT_BY.DEFAULT` please.

Reply:
done